test/memory.test.js

```diff
@@ -169,6 +169,9 @@ describe('Memory connector', function() {
           name: String,
         },
       ],
+      foo: {
```

Comment:
Please use a more descriptive name than `foo`. Can we perhaps move the `date` field to `address` and call it e.g. `since`, meaning the date since the address is valid (when the person moved in)?

Then in tests, you would use `address.since` instead of `foo.date`.